homeassistant/components/comfoair/fan.py

```diff
@@ -0,0 +1,106 @@
+"""Platform to control a Zehnder ComfoAir 350 ventilation unit."""
+import logging
+from typing import Optional
+
+from comfoair.asyncio import ComfoAir
+
+from homeassistant.components.fan import (
+    SPEED_HIGH,
+    SPEED_LOW,
+    SPEED_MEDIUM,
+    SPEED_OFF,
```

Comment:
You're right, but I'd prefer to keep it this way, because SPEED_OFF has special handling inside FanEntity.

I haven't tried it yet, but you could possibly set the fan speed for "AWAY" to 0% (command 0x00CF). The default is around 15% IIRC.

Reply:
Fair enough